README.md

```diff
@@ -1,103 +1,52 @@
 
-## Browser and assistive technology support
+If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
 
-GOV.UK Publishing Components shares the same standards in terms of browser and assistive technology support with [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-and-assistive-technology-support).
-
-## Documentation
-
-[See the rubydoc.info documentation](http://www.rubydoc.info/gems/govuk_publishing_components)
-
-## Working locally
-
-### Install dependencies
-
-As well as the Ruby, you will also need the [Yarn package manager](https://classic.yarnpkg.com/en/docs/install/) to allow installing [npm](https://npmjs.com) dependencies.
-
-Install the application's dependencies with:
+### Running the test suite
 
 ```sh
-bundle install
-yarn install
+bundle exec rake
 ```
 
-### Running a local app with a local gem
-
-The GDS Developer Docs has a [guide on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) that covers setting up a local app to use the local version of the components gem in Docker and using the `./startup.sh` script.
+To run JavaScript tests (only):
 
-### Running tests
-
-The default rake task runs all the linting and test tasks:
-
-```sh
-bin/rake
 ```
-Javascript is tested using Jasmine and the [Jasmine gem](https://github.com/pivotal/jasmine-gem). Tests can be run either in the browser or on the command line via the dummy app’s tasks:
-
-```sh
-# browser
+# run JS tests in browser
 bin/rake app:jasmine
 
-# command line
+# run JS tests on command line
 bin/rake app:jasmine:ci
 ```
 
-### Viewing the component guide
-
-View the component guide in your browser with:
-
-```sh
-./startup.sh
-```
-
-The guide should be available at http://localhost:3212
-
-### View documentation
-
-Preview documentation in your browser with:
-
-```sh
-bundle exec yard server --reload
-```
+### Further documentation
+
+- [Install and use this gem](docs/install-and-use.md)
+- [Use a component in your application](docs/use-components.md)
+- [Generate a new component](docs/generate-a-new-component.md)
+- [Testing a component](docs/testing-components.md)
+- [Move a component from an application to the gem](docs/moving-components-upstream-into-this-gem.md)
+- [Publish/release a new version of the gem](docs/publishing-to-rubygems.md)
+- [Keep this gem in sync with the Design System](docs/upgrade-govuk-frontend.md)
+- [Code documentation on rubydoc.info](http://www.rubydoc.info/gems/govuk_publishing_components)
+* [Component conventions](docs/component_conventions.md)
```

Comment:
```suggestion
- [Component conventions](docs/component_conventions.md)
```

Annoyingly, in GFM if we use a different symbol for lists (in this case `*` instead of `-`) the list will be split into two.